tests/langchain/test_langchain_autolog.py

```diff
@@ -891,5 +892,9 @@ def test_langchain_autolog_callback_injection_in_stream(invoke_arg, generate_con
     expected_logs = ["chain_start", "chain_end"]
     if isinstance(callbacks, BaseCallbackManager):
         assert callbacks.handlers[0].logs == expected_logs
+        assert (
+            sum(isinstance(handler, MlflowLangchainTracer) for handler in callbacks.handlers) == 1
+        )
```

Comment:
This was previously `2`, which was an error.

Technically, we shouldn't be mutating the list of callbacks that the user provides to the `stream()` call. In the interest of time, I'm adjusting this test for now and will file a ticket for fixing the erroneous modification of the user's callback list and adjust the test case